templates/predict_result.html

```diff
@@ -23,6 +23,7 @@
                 <br><br><br><br><br><br><br>
                 <h3>Result of prediction:</h3>
                 <img src="{{ MEDIA_URL }}images/{{ imagefile }}" alt="" height='400'>
+                <img src="{{ MEDIA_URL }}images/{{ imagefilebb }}" alt="" height='400'>
```

Comment:
we don't need templates anymore, these files are redundant.

Reply:
Okay!